src/Altinn.Apps/KubernetesWrapper/Program.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace KubernetesWrapper
+{
+    /// <summary>
+    /// This is the main entry point
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// The main method
+        /// </summary>
+        /// <param name="args">The Arguments</param>
+        public static void Main(string[] args)
+        {
+            CreateWebHostBuilder(args).Build().Run();
+        }
+
+        /// <summary>
+        /// Configure the configuration builder
+        /// </summary>
+        /// <param name="args">arguments for creating build configuration</param>
+        /// <returns>The web host builder</returns>
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
```

Comment:
You should add serilog to the project and instantiate it here, and use it to log error that occurs

Reply:
Fixed.

Reply:
There should also be some serilog stuff in the appsettings json